api/dto/triggers.go

```diff
@@ -195,7 +195,7 @@ func (trigger *Trigger) Bind(request *http.Request) error {
 
 	middleware.SetTimeSeriesNames(request, metricsDataNames)
 
-	if _, err := triggerExpression.Evaluate(); err != nil {
+	if err = triggerExpression.Validate(); err != nil {
```

Comment:
I don't understand this. I think the Validate function should only do one thing. Since Evaluate returns a result we do not need, I think there is no need to call it.

Reply:
I figured this way we maximize the chance of catching a bug if it's in runtime, but overall it's not that big of a deal